tests/integration/tier1/changed-grub-file/main.fmf

```diff
@@ -1,4 +0,0 @@
-summary: changed-grub-file
```

Comment:
Will these deleted tier1 test be reverted too?

Reply:
Not sure, what you mean by reverted?

Reply:
I meant to revert the delete, which you have already done :-)